src/main/resources/assets/create/lang/zh_cn.json

```
@@ -636,153 +667,193 @@
 	"advancement.create.crushing_wheel": "一对大家伙",
 	"advancement.create.crushing_wheel.desc": "制作一些能更快粉碎物品的粉碎轮",
-	"advancement.create.blaze_cake": "糖份超标",
-	"advancement.create.blaze_cake.desc": "给烈焰人燃烧室烤一份特别蛋糕",
-	"advancement.create.wand_of_symmetry": "简单的镜面几何学",
-	"advancement.create.wand_of_symmetry.desc": "制作一根对称之杖",
-	"advancement.create.extendo_grip": "biu biu biu!",
-	"advancement.create.extendo_grip.desc": "获得一个伸缩机械手",
+	"advancement.create.haunted_bell": "黑暗感官",
+	"advancement.create.haunted_bell.desc": "敲响一个缠魂钟",
+	"advancement.create.clockwork_bearing": "巧械时钟",
+	"advancement.create.clockwork_bearing.desc": "组装安装在发条轴承上的装置",
+	"advancement.create.display_link": "大数据",
+	"advancement.create.display_link.desc": "用翻牌显示器可视化资讯",
 	"advancement.create.potato_cannon": "Fwoomp!",
 	"advancement.create.potato_cannon.desc": "用土豆加农炮击杀一个生物",
-	"advancement.create.dual_extendo_grip": "biu——biu——biu——",
-	"advancement.create.dual_extendo_grip.desc": "双持伸缩机械手，超人般的触碰距离",
-	"advancement.create.eob": "Beta 版结束",
-	"advancement.create.eob.desc": "期待未来会有更多的内容 <3",
+	"advancement.create.extendo_grip": "biu biu biu!",
+	"advancement.create.extendo_grip.desc": "获得一个伸缩机械手",
+	"advancement.create.linked_controller": "远程激活",
+	"advancement.create.linked_controller.desc": "用无线红石遥控器激活无线红石终端",
+	"advancement.create.arm_blaze_burner": "熊熊燃烧",
+	"advancement.create.arm_blaze_burner.desc": "指导动力臂给烈焰人燃烧室投食",
+	"advancement.create.crusher_maxed_0000": "压碎它",
+	"advancement.create.crusher_maxed_0000.desc": "让一对粉碎轮以最高速度运行",
+	"advancement.create.arm_many_targets": "掌控全局",
+	"advancement.create.arm_many_targets.desc": "配置一个有十个或更多输出位置的动力臂",
+	"advancement.create.potato_cannon_collide": "蔬菜烟花",
+	"advancement.create.potato_cannon_collide.desc": "让两种土豆发射器弹射物相撞",
+	"advancement.create.self_deploying": "自动驾驶矿车",
+	"advancement.create.self_deploying.desc": "让一个矿车装置在自己前方铺铁轨",
+	"advancement.create.fist_bump": "来碰个拳，哥们~",
+	"advancement.create.fist_bump.desc": "使两个机械手互相碰拳",
+	"advancement.create.crafter_lazy_000": "权宜之计",
+	"advancement.create.crafter_lazy_000.desc": "大幅度减慢动力合成器来延迟基本建设",
+	"advancement.create.extendo_grip_dual": "点满升级",
+	"advancement.create.extendo_grip_dual.desc": "用双重伸缩机械手获得超人般的触及距离§7\n（隐藏进度）",
+	"advancement.create.musical_arm": "没人能在我的 BGM 里打败我！",
+	"advancement.create.musical_arm.desc": "使用动力臂播放唱片",
+	"advancement.create.sturdy_sheet": "坚如磐石",
+	"advancement.create.sturdy_sheet.desc": "用精炼粉碎黑曜石装配坚固薄板",
+	"advancement.create.train_casing_00": "运输时代",
+	"advancement.create.train_casing_00.desc": "用坚固薄板制作列车组件的机壳",
+	"advancement.create.train": "各位请上车!",
+	"advancement.create.train.desc": "装配你的第一辆列车",
+	"advancement.create.conductor": "指挥者的指挥者",
+	"advancement.create.conductor.desc": "使用列车时刻表指挥列车司机",
+	"advancement.create.track_signal": "交通控制",
+	"advancement.create.track_signal.desc": "放置一个列车信号机",
+	"advancement.create.display_board_0": "动态时间表",
+	"advancement.create.display_board_0.desc": "利用显示链接器，在翻牌显示器上预报列车入站时间",
+	"advancement.create.track_0": "新尺度",
```

`更大的新尺度`

---

有点怪

---

`新型铁轨`？

---

> A new Gauge

Gauge 虽然是测量、度量之类的意思，但是在轨道交通上有个更让人熟悉的概念：轨距（可查维基百科）

```suggestion
	"advancement.create.track_0": "新轨距",
```